examples/src/python/example/README.md

```diff
@@ -69,7 +69,7 @@ For example, to use python3 for the whole repo, update `pants.ini` as follows:
 
 ```ini
 [python-setup]
-interpreter_constraints: ["CPython>=3.5"]
+interpreter_constraints: ["CPython>=3.6"]
```

Comment:
It's fine to update it, but this should be unrelated under which python Pants runs.